### DISCUSSION TITLE
Rename sketch in panel by using F2

### DISCUSSION THREAD

**Discussion by Intuos5** [hlorus/CAD_Sketcher#158] [Feature Requests]
**Motivation / Existing Problem**
Currently, we have to edit a sketch to change its name. 

**Description**
Please allow renaming a sketch by pressing F2 after selecting the sketch in the N-panel.

**Alternative Solutions**
An alternative solution is right click to rename and/ or double clicking to rename.

**Comment by hlorus** [comment]
Not quite sure why this isn't working out of the box. Maybe custom [ui lists](https://docs.blender.org/api/current/bpy.types.UIList.html) don't support that. Would be good to know if this works in other addons that have custom ui lists.
